### DISCUSSION TITLE
Is there any plan to support Deno?

### DISCUSSION THREAD

**Discussion by ghost** [fastify/fastify#3753] [Ideas]
I have recently discovered fastify & It's really Cool way of writing web Server. It's Fast, Extensible. I have deployed my simple server to horoku, but what I noticed is that region selection is needed means the servers is running in restricted data center, it's not globally distributed & deployed at the Edge like Cloudflare Workers, Deno Deploy, Fastly, ...

I like Deno. How it's simple fast, performant, browser compatible it's API's are. I am wondering fastify can be compatible with Deno & Deno Deploy?

**Comment by mcollina** [comment]
There are no plans to support Deno atm.

If you are looking for an edge computing solution, check out https://fly.io/.

**Reply by ghost** [reply]
Is there an template to deploy fastify servers to fly.io? If there is send me.

**Reply by mcollina** [reply]
You can just follow https://fly.io/docs/getting-started/node/, just make sure to pass `0.0.0.0` as the `host` when you are calling `listen()`, i.e. `app.listen(process.env.PORT || 3000, '0.0.0.0')`.
